solo_study.md

```diff
@@ -54,7 +54,7 @@ Today you objectives are to put in practice the concepts you learned yesterday.
   - [ ] Link to your solution here (jsbin, gist, or a js file in this repo)
 - [ ] Complete the last 5 tasks of [Arrays Tutorial](https://learn.javascript.info/array)
  - [ ] Link to your solution here
-- [ ] Complete the last 5 tasks of [Array Methods](https://learn.javascript.info/array)
+- [ ] Complete the last 5 tasks of [Array Methods](https://learn.javascript.info/array-methods)
```

Comment:
Good catch on this. This is a fundamental error to the project itself. It would be good to add a comment to the repository that you forked this from, letting them know about this mistake.